### PR TITLE
tidb-in-kubernetes: fix pv deletion

### PR DESCRIPTION
**Pull request by weekface**

### CODE REVIEW

File: dev/tidb-in-kubernetes/reference/configuration/storage-class.md [hunk 1]
```diff
@@ -259,12 +259,22 @@ kubectl apply -f https://raw.githubusercontent.com/pingcap/tidb-operator/master/
 >
 > TiDB Operator 默认会自动将 PD 和 TiKV 的 PV 保留策略修改为 `Retain` 以确保数据安全。
 
-PV 保留策略是 `Retain` 时，如果确认某个 PV 的数据可以被删除，则需要额外设置其保留策略为 `Delete`，此时，只要对应 PVC 被删除，其 PV 会被自动删除并回收。
+PV 保留策略是 `Retain` 时，如果确认某个 PV 的数据可以被删除，需要通过下面的操作来删除 PV 以及对应的数据：
 
-{{< copyable "shell-regular" >}}
+* 首先删除 PV 对应的 PVC 对象：
```
Comment (CaitinChen) [inline dev/tidb-in-kubernetes/reference/configuration/storage-class.md]:
```suggestion
1. 删除 PV 对应的 PVC 对象：
```

File: dev/tidb-in-kubernetes/reference/configuration/storage-class.md [hunk 2]
```diff
@@ -259,12 +259,22 @@ kubectl apply -f https://raw.githubusercontent.com/pingcap/tidb-operator/master/
...
 
-```shell
-kubectl patch pv <pv-name> -p '{"spec":{"persistentVolumeReclaimPolicy":"Delete"}}'
-```
+  {{< copyable "shell-regular" >}}
```
Comment (CaitinChen) [inline dev/tidb-in-kubernetes/reference/configuration/storage-class.md]:
```suggestion
    {{< copyable "shell-regular" >}}
```

File: dev/tidb-in-kubernetes/reference/configuration/storage-class.md [hunk 3]
```diff
@@ -259,12 +259,22 @@ kubectl apply -f https://raw.githubusercontent.com/pingcap/tidb-operator/master/
...
+
+  ```shell
```
Comment (CaitinChen) [inline dev/tidb-in-kubernetes/reference/configuration/storage-class.md]:
```suggestion
    ```shell
```

File: dev/tidb-in-kubernetes/reference/configuration/storage-class.md [hunk 4]
```diff
@@ -259,12 +259,22 @@ kubectl apply -f https://raw.githubusercontent.com/pingcap/tidb-operator/master/
...
+  kubectl delete pvc <pvc-name> --namespace=<namespace>
```
Comment (CaitinChen) [inline dev/tidb-in-kubernetes/reference/configuration/storage-class.md]:
```suggestion
    kubectl delete pvc <pvc-name> --namespace=<namespace>
```

File: dev/tidb-in-kubernetes/reference/configuration/storage-class.md [hunk 5]
```diff
@@ -259,12 +259,22 @@ kubectl apply -f https://raw.githubusercontent.com/pingcap/tidb-operator/master/
...
+  ```
```
Comment (CaitinChen) [inline dev/tidb-in-kubernetes/reference/configuration/storage-class.md]:
```suggestion
    ```
```

File: dev/tidb-in-kubernetes/reference/configuration/storage-class.md [hunk 6]
```diff
@@ -259,12 +259,22 @@ kubectl apply -f https://raw.githubusercontent.com/pingcap/tidb-operator/master/
...
+
+* 然后，设置 PV 的保留策略为 `Delete`，PV 会被自动删除并回收：
```
Comment (CaitinChen) [inline dev/tidb-in-kubernetes/reference/configuration/storage-class.md]:
```suggestion
2. 设置 PV 的保留策略为 `Delete`，PV 会被自动删除并回收：
```

File: dev/tidb-in-kubernetes/reference/configuration/storage-class.md [hunk 7]
```diff
@@ -259,12 +259,22 @@ kubectl apply -f https://raw.githubusercontent.com/pingcap/tidb-operator/master/
...
+
+  {{< copyable "shell-regular" >}}
```
Comment (CaitinChen) [inline dev/tidb-in-kubernetes/reference/configuration/storage-class.md]:
```suggestion
    {{< copyable "shell-regular" >}}
```

File: dev/tidb-in-kubernetes/reference/configuration/storage-class.md [hunk 8]
```diff
@@ -259,12 +259,22 @@ kubectl apply -f https://raw.githubusercontent.com/pingcap/tidb-operator/master/
...
+
+  ```shell
```
Comment (CaitinChen) [inline dev/tidb-in-kubernetes/reference/configuration/storage-class.md]:
```suggestion
    ```shell
```

File: dev/tidb-in-kubernetes/reference/configuration/storage-class.md [hunk 9]
```diff
@@ -259,12 +259,22 @@ kubectl apply -f https://raw.githubusercontent.com/pingcap/tidb-operator/master/
...
+  kubectl patch pv <pv-name> -p '{"spec":{"persistentVolumeReclaimPolicy":"Delete"}}'
```
Comment (CaitinChen) [inline dev/tidb-in-kubernetes/reference/configuration/storage-class.md]:
```suggestion
    kubectl patch pv <pv-name> -p '{"spec":{"persistentVolumeReclaimPolicy":"Delete"}}'
```

File: dev/tidb-in-kubernetes/reference/configuration/storage-class.md [hunk 10]
```diff
@@ -259,12 +259,22 @@ kubectl apply -f https://raw.githubusercontent.com/pingcap/tidb-operator/master/
...
+  ```
```
Comment (CaitinChen) [inline dev/tidb-in-kubernetes/reference/configuration/storage-class.md]:
```suggestion
    ```
```

**Comment by weekface** [comment]
comments addressed @CaitinChen PTAL

**Comment by CaitinChen** [comment]
@TomShawn Please help align this doc in the docs repo. Thanks.

**Comment by TomShawn** [comment]
> @TomShawn Please help align this doc in the docs repo. Thanks.

Sure.

**Pull request merged**